{"resultsPerPage": 1, "startIndex": 0, "totalResults": 1, "format": "NVD_CVE", "version": "2.0", "timestamp": "2023-06-17T19:32:56.457", "vulnerabilities": [{"cve": {"id": "CVE-2022-40899", "sourceIdentifier": "cve@mitre.org", "published": "2022-12-23T00:15:14.110", "lastModified": "2023-01-23T18:57:18.740", "vulnStatus": "Analyzed", "descriptions": [{"lang": "en", "value": "An issue discovered in Python Charmers Future 0.18.2 and earlier allows remote attackers to cause a denial of service via crafted Set-Cookie header from malicious web server."}], "metrics": {"cvssMetricV31": [{"source": "nvd@nist.gov", "type": "Primary", "cvssData": {"version": "3.1", "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H", "attackVector": "NETWORK", "attackComplexity": "LOW", "privilegesRequired": "NONE", "userInteraction": "NONE", "scope": "UNCHANGED", "confidentialityImpact": "NONE", "integrityImpact": "NONE", "availabilityImpact": "HIGH", "baseScore": 7.5, "baseSeverity": "HIGH"}, "exploitabilityScore": 3.9, "impactScore": 3.6}]}, "weaknesses": [{"source": "nvd@nist.gov", "type": "Primary", "description": [{"lang": "en", "value": "NVD-CWE-noinfo"}]}], "configurations": [{"nodes": [{"operator": "OR", "negate": false, "cpeMatch": [{"vulnerable": true, "criteria": "cpe:2.3:a:pythoncharmers:python-future:*:*:*:*:*:python:*:*", "versionEndIncluding": "0.18.2", "matchCriteriaId": "94A8576C-172D-435B-9A08-9811AED64A47"}]}]}], "references": [{"url": "https://github.com/PythonCharmers/python-future/blob/master/src/future/backports/http/cookiejar.py#L215", "source": "cve@mitre.org", "tags": ["Third Party Advisory"]}, {"url": "https://github.com/PythonCharmers/python-future/pull/610", "source": "cve@mitre.org", "tags": ["Issue Tracking", "Patch", "Third Party Advisory"]}, {"url": "https://github.com/python/cpython/pull/17157", "source": "cve@mitre.org", "tags": ["Exploit", "Patch", "Third Party Advisory"]}, {"url": "https://pypi.org/project/future/", "source": "cve@mitre.org", "tags": ["Product", "Third Party Advisory"]}, {"url": "https://pyup.io/posts/pyup-discovers-redos-vulnerabilities-in-top-python-packages/", "source": "cve@mitre.org", "tags": ["Exploit", "Vendor Advisory"]}]}}]}